odoo/tools/config.py

```diff
@@ -397,8 +398,11 @@ def die(cond, msg):
         # Ensures no illegitimate argument is silently discarded (avoids insidious "hyphen to dash" problem)
         die(args, "unrecognized parameters: '%s'" % " ".join(args))
 
-        die(bool(opt.syslog) and bool(opt.logfile),
-            "the syslog and logfile options are exclusive")
+        die(bool(opt.syslog) + bool(opt.logfile) + bool(opt.logfilesdb) > 1,
+            "the syslog, logfile and logfilesdb options are exclusive")
+
+        die(opt.logfilesdb and '%db' not in os.path.basename(opt.logfilesdb),
+            "the logfilesdb option must contain the '%db' pattern in the file name")
```

Comment:
This only test for the cli argument and not the configuration from the configuration file. Also now that we have a unittest class to test all stuff related to the config, please add a test to cover this one, and also the `die(bool(opt.syslog) + bool(opt.logfile) + bool(opt.logfilesdb) > 1,` bit

Reply:
hummm I don't think we have any `die()` test yet, you'll need to mock `config.parser.error` and assert it was called with the given error message

Reply:
I think it's even worst than that.
Already without my PR If I define for example `syslog=True` in the config file and `logfile` as cli parameter. It will not raise any error. It will log in `syslog` in this case due to the `if syslog elif logfile` in the code.
I guess I will fix this issue as well

Reply:
Indeed, let's just write a test for `die(opt.logfilesdb and '%db' not in os.path.basename(opt.logfilesdb),` and leave the other one buggy. There are tons of bugs inside this module and I've fixed half of them inside a PR of mine that is rotting somewhere.

Reply:
I'm a bit confused by your last message, I updated the code like I think you suggested. But in this case it would die only in the case where the CLI parmaeter is used. As a config file parameter it will just ignore. Am I missing something to your suggestion ?